phpBB/includes/functions_admin.php

```diff
@@ -642,7 +642,7 @@ function move_posts($post_ids, $topic_id, $auto_sync = true)
 	 * @event core.move_posts_before
 	 * @var	array	post_ids	Array of post ids to move
 	 * @var	string	topic_id	The topic id the posts are moved to
-	 * @var bool	auto_sync	Whether or not to perform auto sync
```

Comment:
The file was not consistent with its tab/space characters, so I corrected it.  I'll remember to include that in the commit message next time.